packages/react-ui-components/src/stories/Components/StateIndicator.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvStateIndicator } from '../../components';
+
+KvStateIndicator.displayName = 'KvStateIndicator';
+
+export default {
+	title: "Components/StateIndicator",
+	component: 'kv-state-indicator',
+	argTypes: {
+		color: { type: "color" },
+		text: { type: "text" }
+	},
+	parameters: {
+		notes: require('../../../../ui-components/src/components/state-indicator/readme.md')
```

Comment:
please rebase with dev branch and update to require('@ui-notes/state-indicator/readme.md')